designer/server/src/models/account/auth.js

```diff
@@ -1,5 +1,27 @@
+import config from '~/src/config.js'
+
 export function signedOutViewModel() {
   return {
     pageTitle: 'You have signed out'
   }
 }
+
+/**
+ * @param {boolean} userFailedAuthorisation - indicates the user has failed authorisation, but authentication was successful
+ */
+export function signInViewModel(userFailedAuthorisation) {
+  const errorList = []
+
+  if (userFailedAuthorisation) {
+    errorList.push({
+      html: `We couldn’t sign you in. Please contact the system administrator for help,
```

Comment:
Mind double checking if this is correct?

Usually avoid [negative contractions](https://www.gov.uk/service-manual/design/writing-for-user-interfaces#contractions-abbreviations-and-acronyms) like "couldn't"

```suggestion
      html: `We could not sign you in. Please contact the system administrator for help,
```

Reply:
I lifted it from Figma, I'll check with Dan.

Reply:
Changed approved by Dan. Fixed.